src/bootstrap.ts

```diff
@@ -58,8 +78,10 @@ import { logger } from './logging';
 import { ProcessEnvironmentDetector } from './resources/detectors/ProcessEnvironmentDetector';
 import { LumigoSpanProcessor } from './resources/spanProcessor';
 import { getLumigoSampler } from './samplers/lumigoSampler';
+import { LumigoLogRecordProcessor } from './processors/LumigoLogRecordProcessor';
 
-const lumigoEndpoint = process.env.LUMIGO_ENDPOINT || DEFAULT_LUMIGO_ENDPOINT;
+const lumigoTraceEndpoint = process.env.LUMIGO_ENDPOINT || DEFAULT_LUMIGO_TRACES_ENDPOINT;
```

Comment:
We need to discuss the best approach to this, as I've covered in a [DR](https://www.notion.so/14-02-2024-DR-Auto-Instrumentation-Endpoint-via-Operator-35a121f946c7481c847992ae703dd03c), we already have a mis-alignment with env var use.

Without changes to the operator, a custom env var for a logs endpoint will not be set and auto-instrumentation for logs would be sent directly to the backend and skip the collector in the operator. This would lead to data not being enriched with information from the cluster/namespace

Reply:
Note that this only changes the name of a constant, not an env-var - I'm still using `LUMIGO_ENDPOINT` for constructing the full URL of the logs-endpoint as well, so the operator should be good with this - am I missing something?